app/index.js

```diff
@@ -55,6 +55,7 @@ const {resolve} = require('path');
 const {app, BrowserWindow, Menu} = require('electron');
 const {gitDescribe} = require('git-describe');
 const isDev = require('electron-is-dev');
+const WindowStateManager = require('electron-window-state-manager');
```

Comment:
Is this only used to save and restore if the window was maximized....?

Reply:
If yes, why can't this be in a plugin like https://www.npmjs.com/package/hyperfullscreen or https://github.com/simonmeusel/hyperfull 🤔

Reply:
the ones u mentioned here are to make it full screen. whereas this code will make it dynamic depending on the clients need, whether he wants the app to be full screen partially filled or any custom dimension that he wants. This will store the data from the previous interaction with hyper and apply the same to a new instance when created.